apis/interfaces.go

```diff
@@ -35,6 +35,16 @@ type Validatable interface {
 	Validate(context.Context) *FieldError
 }
 
+// Convertible indicates that a particular type supports conversions to/from
+// "higher" versions of the same type.
+type Convertible interface {
```

Comment:
Do we care to specify whether we're doing staged a⇒b⇒c or a⇒c, or note that how a is converted to c is not prescribed and is an implementation detail?

Reply:
I think that beyond the direction it's an implementation detail for you get from `a` to `c`.

Reply:
Well, then it would make sense to say, that any version above/below `x` is acceptable